plugin/src/shared/sc/plugin2021/util/GameRuleLogic.kt

```
@@ -72,32 +67,22 @@ object GameRuleLogic {
      * @return gibt einen [MoveMistake] zurück, wenn der Zug nicht valide wahr, ansonsten null
```

```suggestion
     * @return ein [MoveMistake], wenn der Zug nicht valide ist, ansonsten null
```

---

wait this isn't even accurate anymore - it throws now instead of returning!

---

oh yeah, forgot to update the docs